code/game/objects/items/weapons/tools.dm

```diff
@@ -13,6 +13,15 @@
  */
 
 /*
+ * Used for fancy tool subtypes that are faster or slower than the standard tool.
+ * The number in the first slot is the multiplier for time to construct, the one in the second slot is the multiplier for time to deconstruct.
+ * If one is zero, the tool cannot be used in that direction. If you want to adminbus an instant tool, use .0001 or something, not 0.
+ * Don't set either to a negative number. It will probably break, though I'm not really sure in what way.
+ */
+/atom
+	var/list/construction_delay_mult = list(1, 1)

```

Comment:
Something like

`list(Co_CON_SPEED = 1, Co_DECON_SPEED = 1)`

The general rule is that a define is impossible to make a typo in, because the compiler throws an error. Just having a string, and using that string in the code, risks you making a mistake somewhere.
The Co prefix is used on all construction defines.